db/migrate/20160229131112_add_identifier_to_payments.rb

```diff
@@ -0,0 +1,5 @@
+class AddIdentifierToPayments < ActiveRecord::Migration
+  def change
```

Comment:
`identifier` is being used in `Spree::AmazonController#payment` action and it is used to show payer_id on payments page admin end